ci/script.sh

```diff
@@ -23,9 +18,42 @@ inspect() {
     set +e
     $PREFIX$OBJDUMP -Cd target/**/release/*.rlib | gist_it
     set -e
+}
+
+run_tests() {
+    if [[ $QEMU_LD_PREFIX ]]; then
+        export RUST_TEST_THREADS=1
+    fi
+
+    if [[ $RUN_TESTS == y ]]; then
+        cargo test --target $TARGET
+        cargo test --target $TARGET --release
+    fi
+}
+
+c_test() {
+    $CARGO clean
+
+    # verify we haven't drop any intrinsic/symbol
+    $CARGO build --features c --target $TARGET --bin intrinsics
```

Comment:
Would it make sense to have `RUSTFLAGS="-C link-args='-Wl,-nostartfiles -Wl,-nodefaultlibs'"` so that it applies to all targets? I'm assuming that we don't need libc anywhere in there...